Example/CollectionViewToolsExample/Diff/CollectionViewDiffs/CollectionViewIGListKitDiff.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright © 2019 Rosberry. All rights reserved.
+//
+
+import IGListKit
+import CollectionViewTools
+
+final class CollectionViewIGListKitDiff: CollectionViewDiff {
+
+    func changes<T: DiffItem>(old: [T], new: [T]) -> [CollectionViewChange<T>] {
+        let oldWrappers = old.map { IGListKitDiffableItemWrapper(item: $0) }
+        let newWrappers = new.map { IGListKitDiffableItemWrapper(item: $0) }
+        let result = ListDiff(oldArray: oldWrappers, newArray: newWrappers, option: .equality)
+        let inserts = result.inserts.map { index in
+            CollectionViewChange(insert: .init(item: new[index], index: index))
+        }
+        let deletes = result.deletes.map { index in
+            CollectionViewChange(delete: .init(item: old[index], index: index))
+        }
+        let moves = result.moves.map { move in
+            CollectionViewChange(move: CollectionViewMove(item: new[move.to],
+                                                          from: move.from,
+                                                          to: move.to))
+        }
+        let updates = result.updates.map { index -> CollectionViewChange<T> in
+            let oldItem = old[index]
+            var newItem: T?
+            var newIndex: Int?
+            for (index, item) in new.enumerated() {
+                guard oldItem.diffIdentifier == item.diffIdentifier else {
+                    continue
+                }
+                newItem = item
+                newIndex = index
+                break
+            }
+            return CollectionViewChange(update: CollectionViewUpdate(oldItem: oldItem,
+                                                                     newItem: newItem,
+                                                                     index: newIndex))
+        }
+        let changes = inserts + deletes + updates + moves
+        return changes
+    }
+}
+
+public final class IGListKitDiffableItemWrapper<T: DiffItem>: ListDiffable {
+
+    let item: T
+
+    init(item: T) {
+        self.item = item
+    }
+
+    public func diffIdentifier() -> NSObjectProtocol {
```

Comment:
Think will better if it will looks like computed property

Reply:
What do you mean? It's a function from ListDiffable protocol in IGListKit

Reply:
Ok, sorry, I miss it